### ISSUE TITLE
Editions table on book pages contains link to the current page

### ISSUE THREAD

**Issue by jimchamp** [internetarchive/openlibrary#6040]
One primary function of the editions table is to present other editions of the book that a patron is currently viewing.  Today, the first item in the editions table references the page that the table is on (more clearly: clicking on the first link in the table reloads the current book page).  Aside from a slight color difference, there is nothing that indicates that the first item in the table is the book currently being viewed by the patron.  Removing this item should make it slightly easier for patrons to quickly find other, more suitable, editions.

### Relevant url?

https://openlibrary.org/books/OL7824010M/Where_the_Red_Fern_Grows or any other book page

### Steps to Reproduce

1. Go to a book page.
2. Click the link in the first item of the editions table.

* Actual: The page reloads, leaving me somewhat confused...
* Expected: A different book page is loaded.

### Details

- **Logged in (Y/N)?** N
- **Browser type/version?** N/A
- **Operating system?** N/A
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints

When table rows are being rendered, prevent the row that references the current page from being rendered.

### Related files

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/templates/type/work/editions_datatable.html

### Stakeholders

@Hitansh-Shah

**Comment by jimchamp** [comment]
@Hitansh-Shah I don't think that I am able to assign this to you until you comment on this issue.  If you're still interested in working on this, please comment and I will assign you.  Thanks!

**Comment by Hitansh-Shah** [comment]
> @Hitansh-Shah I don't think that I am able to assign this to you until you comment on this issue. If you're still interested in working on this, please comment and I will assign you. Thanks!

Yup I will work on this issue :)

**Comment by Hitansh-Shah** [comment]
@jimchamp For local environment I imported test data locally according to this [guide](https://github.com/internetarchive/openlibrary/wiki/Getting-Started#importing-test-data). However it imports only one edition. I also followed [this](https://github.com/internetarchive/openlibrary/wiki/Getting-Started#:~:text=To%20import%20a%20work%27s%20editions) but it gave `404` error. So how do I import all editions in order to work on editions table?

**Comment by agmckee** [comment]
_Three Cups of Tea_ (ISBN: 9780142414125) in the sample data has multiple editions and I was also able to import additional editions using that snippet of JS and the resulting copydocs snippet.

**Comment by Hitansh-Shah** [comment]
> _Three Cups of Tea_ (ISBN: 9780142414125) in the sample data has multiple editions and I was also able to import additional editions using that snippet of JS and the resulting copydocs snippet.

It worked😇! Thanks a lot. However do you have any idea why it is not working for _Where_the_Red_Fern_Grows_(OL7824010M) ?

**Comment by agmckee** [comment]
I don't know, sorry. Might be worth collecting some information on that and reporting it as a new issue, as I believe it should work for any work. Glad my suggestion worked out for you though and hopefully allows you to get started on this issue.

**Comment by Hitansh-Shah** [comment]
> I don't know, sorry. Might be worth collecting some information on that and reporting it as a new issue, as I believe it should work for any work. Glad my suggestion worked out for you though and hopefully allows you to get started on this issue.
> 
> Edit to add: I was able to import OL7824010M into my local environment. I dropped into the web container `docker-compose exec web bash` and imported the book using this command: `./scripts/copydocs.py /works/OL465360W --recursive`. Note that you import the work, not the specific edition.

Ok got it 👍️. Thanks a lot for guiding 😄

**Comment by mekarpeles** [comment]
@mheiman is a stakeholder in this one who suggested the feature. Assuming we go in this direction, the code is broadly right.

There are some edge cases / open questions @jimchamp 

1, What about book pages with only 1 edition? i.e. What do we do if there are no other editions?
2. We may want to change the nav bar to say e.g. "(n-1) Other Editions" rather than View 4 Editions

**Comment by Hitansh-Shah** [comment]
Question 2 sounds reasonable to me. For question 1 we can display something like "there are no other editions for this book" type statement to help readers know. In addition to that we can also add a section for the case where there is another edition but we are not able to show for some reason. Therefore in that section we can provide users a way to tell us about this.

**Comment by jimchamp** [comment]
This is not a bug --- rather, this is how the editions table functions.  Closing for now, but anybody can reopen this is they disagree.
